oauth2client/contrib/django_util/__init__.py

```diff
@@ -210,6 +270,23 @@ def _get_oauth2_client_id_and_secret(settings_instance):
                 "GOOGLE_OAUTH2_CLIENT_SECRET in settings.py")
 
 
+def _get_storage_model():
+    """This configures whether the credentials will be stored in the session
```

Comment:
To address your original question that I still have in my email: every code element, even private code elements, _potentially_ has users who are not the same individuals as its maintainers. In practice they're usually the same for private code elements in small modules. The principle for implementation comment versus doc string still applies though: is the information you're relating germane to _use_ of the code element or _maintenance_ of it?